extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/jaxrs/ResteasyReactiveViolationExceptionMapper.java

```diff
@@ -56,17 +64,25 @@ private boolean isReturnValueViolation(ConstraintViolation<?> violation) {
         return secondNode.getKind() == ElementKind.RETURN_VALUE;
     }
 
-    protected Response buildResponse(Object entity, Status status) {
-        return Response.status(status).entity(entity).build();
-    }
-
     private Response buildViolationReportResponse(ConstraintViolationException cve) {
+        Status status = Status.BAD_REQUEST;
+        Response.ResponseBuilder builder = Response.status(status);
+        builder.header(Validation.VALIDATION_HEADER, "true");
+
+        // Check standard media types.
+        MediaType mediaType = ValidatorMediaTypeUtil.getAcceptMediaTypeFromSupported(headers.getAcceptableMediaTypes());
```

Comment:
Wasn't this the one that did a bunch of reflection?

Reply:
No, this method is simply checking whether the list of media items provided by the user via the ACCEPT header is supported. 
The method that was using reflection was removed from this util.